File: sdi2-13/src/com/sdi/model/UserLogin.java
package com.sdi.model;

import java.io.Serializable;

public class UserLogin implements Serializable{
	private static final long serialVersionUID = 1L;
	
	public UserLogin(String login, String name, Long id) {
		super();
		this.login = login;
		this.name = name;
		this.setId(id);
	}
	private String login;
	private String name;
	private Long id;
	
	public String getLogin() {
		return login;
	}
	public void setLogin(String login) {
		this.login = login;
	}
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public Long getId() {
		return id;
	}
	public void setId(Long id) {
		this.id = id;
	}

	
}

File: sdi3-13.Cli-REST_CS/sdi3-13.Cli-REST_CSharp/model/user/UserLogin.cs

public class UserLogin{
	private static readonly long serialVersionUID = 1L;
	
	public UserLogin(string login, string name, long id) {
		this.login = login;
		this.name = name;
		this.setId(id);
	}
	private string login;
	private string name;
	private long id;
	
	public string getLogin() {
		return login;
	}
	public void setLogin(string login) {
		this.login = login;
	}
	public string getName() {
		return name;
	}
	public void setName(string name) {
		this.name = name;
	}
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}

	
}
